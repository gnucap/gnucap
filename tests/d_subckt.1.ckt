'
V1   2  0  1.
X1   2  1  3 zzz
X2   2  4  5 zzz
.subckt zzz  1 2 4
R1   1  2  1.
R2   2  0  1.
R3   2  3  1.
R4   3  0  1.
r5 3 4 1
r6 4 0 1
.ends zzz
.width out=160
.print op v(nodes)
.op
.print op + v(r1.x1) v(r?.x1) v(r2.x?)
.op
.print
.list
*>.status notime
.end
