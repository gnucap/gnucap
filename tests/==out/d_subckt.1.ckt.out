' 
#           v(1)       v(2)       v(3)       v(4)       v(5)      
 27.        0.38461    0.99999    0.076922   0.38461    0.076922  
#           v(1)       v(2)       v(3)       v(4)       v(5)       v(X1.R1)   v(X1.R1)   v(X1.R2)   v(X1.R3)   v(X1.R4)   v(X1.r5)   v(X1.r6)   v(X1.R2)  
+ v(X2.R2)  
 27.        0.38461    0.99999    0.076922   0.38461    0.076922   0.61538    0.61538    0.38461    0.23077    0.15384    0.076922   0.076922   0.38461   
+ 0.38461   
tran   
ac     
dc     
op      v(1) v(2) v(3) v(4) v(5) v(X1.R1) v(X1.R1) v(X1.R2) v(X1.R3) v(X1.R4) v(X1.r5) v(X1.r6) v(X1.R2) v(X2.R2)
fourier
V1 ( 2 0 )  DC  1.
X1 ( 2 1 3 )  zzz 
X2 ( 2 4 5 )  zzz 
.subckt zzz ( 1 2 4 )
R1 ( 1 2 )  1.
R2 ( 2 0 )  1.
R3 ( 2 3 )  1.
R4 ( 3 0 )  1.
r5 ( 3 4 )  1.
r6 ( 4 0 )  1.
.ends zzz
Gnucap   System status
iterations: op=4, dc=0, tran=0, fourier=0, total=7
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=5, subckt=2, model=0, total=7
dctran density=75.5%, ac density=75.5%
